src/components/Workspace.js

```diff
@@ -101,6 +104,24 @@ const toolbox = `
             </block>
           </value>
         </block>
+        <block type="headlight"></block>
+        <block type="colour_rgb">
+          <value name="RED">
+            <shadow type="math_number">
+              <field name="NUM">100</field>
+            </shadow>
+          </value>
+          <value name="GREEN">
+            <shadow type="math_number">
+              <field name="NUM">50</field>
+            </shadow>
+          </value>
+          <value name="BLUE">
+            <shadow type="math_number">
+              <field name="NUM">0</field>
+            </shadow>
+          </value>
+        </block>
```

Comment:
There's also a cool color-picker block that Blockly provides. But we could throw that in the toolbox in a later PR.